tide_station_locator.module

```diff
@@ -30,28 +31,62 @@ function tide_station_locator_get_api_data($delta = FALSE) {
 /**
  * Function to migrate stations.
  */
-function tide_station_locator_migrate_stations($options) {
+function tide_station_locator_migrate_stations() {
 
   try {
     // Connect to API and download the latest station JSON and migrate.
-    tide_station_locator_get_api_data($options['delta']);
-    if ($options['execute-dependencies'] == TRUE) {
-      $dependent_migrations =
-        [
-          'station_accessibility_terms',
-          'station_speciality_services_facility_terms',
-          'station_state_terms',
-        ];
-      foreach ($dependent_migrations as $migration_ids) {
-        _tide_station_locator_migrate($migration_ids);
-      }
-    }
+    tide_station_locator_get_api_data();
     _tide_station_locator_migrate('station_nodes');
```

Comment:
Why do we need this? `_tide_station_locator_migrate` seems to be equivalent to `drush mim station_nodes`.

Reply:
@vincent-gao Updated the code.